ui/packages/catalog-ui-search/src/main/webapp/test/mock-api/mock-properties.js

```diff
@@ -0,0 +1,26 @@
+const properties = require('../../js/properties')
+const api = require('./index')
+const oldInit = properties.init
+
+const mock = () => {
+  properties.init = function() {
+    const data = api('./internal/config')
+    const uiConfig = api('./internal/platform/config/ui')
+    // use this function to initialize variables that rely on others
```

Comment:
Is this comment still needed?

Reply:
I think it's still useful as to clarify the mandatory nature of this initialization.